cf-agent/package_module.c

```diff
@@ -1220,32 +1246,42 @@ PromiseResult RepoInstall(EvalContext *ctx,
             }
             else
             {
-                if (action == cfa_warn || DONTDO)
+                PromiseResult result = PROMISE_RESULT_FAIL;
+                if (MakingChanges(ctx, pp, attr, &result, "install repo type package: %s",
+                                  package_name))
                 {
-                    Log(LOG_LEVEL_VERBOSE, "Should install repo type package: %s",
-                        package_name);
-                    res = PromiseResultUpdate(res, PROMISE_RESULT_WARN);
+                    if (ChrootChanges())
+                    {
+                        RecordPkgOperationInChroot(CHROOT_PKG_OPERATION_INSTALL, package_name,
+                                                   update_version, update_package->arch);
+                    }
+                    else
+                    {
+                        /* Append package data to buffer. At the end all packages
+                         * data that need to be updated will be sent to package module
+                         * at once. This is important if we have package in more than
```

Comment:
Maybe "simultaneously" instead of "at once" ?
Also "package" should probably be in plural -> "packages" ?

Reply:
or "a package" maybe ?